integration-tests/emulated/chains/parachains/integritee/integritee-kusama/Cargo.toml

```diff
@@ -2,7 +2,7 @@
 name = "integritee-kusama-emulated-chain"
 authors.workspace = true
 edition.workspace = true
-version = "1.18.0"
+version = "1.18.2"
```

Comment:
Shall we use the `version.workspace = true` too, and only manage the runtime separately?